redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 2.2.8
+version: 2.2.9
```

Comment:
From semver perspective, it feels like this update should move `2.2.8` to `2.3.0`. But I assume we are trying to keep the major and minor numbers in sync with Redpanda, so we just increment patch.

Maybe we should come up with a plan and include it in some README. Incrementing patch on any chart change that doesn't impact the default Redpanda version installed would be one way to tell at a glance if the chart should work with a given Redpanda version. But it breaks semver, which in itself is useful.

Reply:
I'm leaning towards following semver (and making this version `2.3.0`).

Reply:
Yeah I could go either way on this. TBH this does probably skew towards "new feature" rather than "bug fix"